src/resolve/resolved-manager.c

```diff
@@ -468,16 +468,14 @@ static int manager_sigusr1(sd_event_source *s, const struct signalfd_siginfo *si
 
 static int manager_sigusr2(sd_event_source *s, const struct signalfd_siginfo *si, void *userdata) {
         Manager *m = userdata;
-        DnsScope *scope;
 
         assert(s);
         assert(si);
         assert(m);
 
-        LIST_FOREACH(scopes, scope, m->dns_scopes)
-                dns_cache_flush(&scope->cache);
-
+        manager_flush_caches(m);
         log_info("Flushed all caches.");
```

Comment:
I think this line should go into manager_flush_caches. It's better to have the same reporting in the logs no matter if the action was invoked by signal or over dbus.